lib/WWW/Mechanize.pm

```diff
@@ -552,15 +540,43 @@ sub head {
     my $self = shift;
     my $uri = shift;
 
+    $uri = $self->_uri_with_base( $uri );
+
+    # It appears we are returning a super-class method,
+    # but it in turn calls the request() method here in Mechanize
+    return $self->SUPER::head( $uri->as_string, @_ );
+}
+
+=head2 $mech->delete ($uri )
```

Comment:
Please indent this correctly. This is important in case we link to it from somewhere else in the documentation.

Reply:
Are you referring to the space? I'm seeing the same pattern at `=head2 $mech->head ($uri )
`

Reply:
Yes, I was. Hmm. I have been through this POD a few times to try to fix all of the inconsistencies, but have missed that. Ok, I think we take it anyway and I can do another pass after that. Thanks @oalders